src/test/java/uk/gov/ch/oauth/tokens/OAuthTokenTest.java

```diff
@@ -14,23 +22,75 @@
     public static final String TOKEN = "TOKEN";
     public static final String TOKEN_TYPE = "TOKEN_TYPE";
 
-    @Test
-    void saveAccessToken() {
-        OAuthToken token = new OAuthToken();
+    private final ObjectMapper mapper = new ObjectMapper();
+    private OAuthToken token;
+
+    @BeforeEach
+    public void setup() {
+        token = new OAuthToken();
 
         token.setExpiresIn(EXPIRES_IN);
         token.setRefreshToken(REFRESH_TOKEN);
         token.setToken(TOKEN);
         token.setTokenType(TOKEN_TYPE);
+    }
 
-        Map<String, Object> map = token.saveAccessToken();
+    @Test
+    @DisplayName("Check access token serailises with correct keys and values")
+    void accessTokenSerialises() throws JsonProcessingException {
+        String serialised = mapper.writeValueAsString(token);
+        JsonNode tokenAsTree = mapper.readValue(serialised, JsonNode.class);
 
-        Map<String, Object> data = (Map<String, Object>) map.get(SessionKeys.ACCESS_TOKEN.getKey());
-        assertNotNull(data);
+        final Iterator<Entry<String, JsonNode>> fields = tokenAsTree.fields();
+        AtomicInteger fieldCount = new AtomicInteger();
+        fields.forEachRemaining(entry -> {
+            checkNodeValue(entry, token);
+            fieldCount.getAndIncrement();
+        });
+        assertEquals(4, fieldCount.get());
+    }
 
-        assertEquals(EXPIRES_IN, data.get(SessionKeys.EXPIRES_IN.getKey()));
-        assertEquals(REFRESH_TOKEN, data.get(SessionKeys.REFRESH_TOKEN.getKey()));
-        assertEquals(TOKEN, data.get(SessionKeys.ACCESS_TOKEN.getKey()));
-        assertEquals(TOKEN_TYPE, data.get(SessionKeys.TOKEN_TYPE.getKey()));
+    @Test
+    @DisplayName("Check AccessToken deserialises correctly.")
+    void setAccessTokenDeserialises() throws JsonProcessingException {
+        String input = "{\"access_token\":\"TOKEN\",\"expires_in\":1,\"refresh_token\":\"REFRESH_TOKEN\",\"token_type\":\"TOKEN_TYPE\"}";
```

Comment:
Use `EXPIRES_IN` instead of 1

Reply:
you mean it should look like?
`\"expires_in\":\"EXPIRES_IN\",`?